pallets/dkg-proposal-handler/src/lib.rs

```diff
@@ -356,7 +350,7 @@ impl<T: Config> ProposalHandlerTrait for Pallet<T> {
 			Self::deposit_event(Event::<T>::ProposalSigned {
 				chain_id,
 				key: DKGPayloadKey::EVMProposal(nonce),
-				data,
+				data: data.clone(),
```

Comment:
Should we try to avoid cloning `data`?

Reply:
Can't do otherwise. data is a `&Vec<T>` here.